### PR TITLE
Create appveyor.yml

### PR DESCRIPTION
**Pull request by wannaphongcom**
Add appveyor

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+build: off
+image: Visual Studio 2015
+
+environment:
+  global:
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
+  matrix:
+#    - PYTHON: "C:/Python36"
+#      PYTHON_VERSION: "3.6"
+#      PYTHON_ARCH: "32"
```
Comment (heytitle) [inline appveyor.yml]: could you please remove this commented code?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    - PYTHON: "C:/Python36-x64"
+      PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "64"
+      DISTUTILS_USE_SDK: "1"
+
+#    - PYTHON: "C:/Python37"
+#      PYTHON_VERSION: "3.7"
+#      PYTHON_ARCH: "32"
+
```
Comment (heytitle) [inline appveyor.yml]: this one as well

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+platform:
+  - x64
+
+install:
+  - "chcp 65001"
+  - "set PYTHONIOENCODING=utf-8"
+  - "%PYTHON%\\python.exe -m pip install wheel"
+#  - ECHO "Installed SDKs:"
+#  - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
+  - IF "%ARCH%"=="32" (call "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" x86) ELSE (ECHO "probably a 64bit build")
```
Comment (heytitle) [inline appveyor.yml]: can we remove these two lines above?

File: appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  - IF "%ARCH%"=="64" (call "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" amd64) ELSE (ECHO "probably a 32bit build")
+  - '"%VS140COMNTOOLS%\..\..\VC\vcvarsall.bat" %PLATFORM%'
+  - ps: if (-not(Test-Path($env:PYTHON))) { & appveyor\install.ps1 }
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "python --version"
+  # - "set ICU_VERSION=62"
```
Comment (heytitle) [inline appveyor.yml]: also here

Comment (wannaphongcom) [reply appveyor.yml]: ok.

File: appveyor.yml (hunk 3 above)
Comment (wannaphongcom) [reply appveyor.yml]: ok

**Comment by wannaphongcom** [comment]
Done

**Pull request merged**